coretex/cli/modules/node.py

```diff
@@ -62,5 +79,61 @@ def shouldUpdate(repository: str, tag: str) -> bool:
             if repository in digest and manifestJson["Descriptor"]["digest"] in digest:
                 return False
         return True
-    except:
-        return False
+    except CommandException:
+        return True
+
+
+def registerNode(name: str) -> str:
+    params = {
+        "machine_name": name
+    }
+    response = networkManager.post("service", params)
+
+    if response.hasFailed():
+        print(response.getJson(dict))
+        raise Exception("Failed to configure node. Please try again...")
+
+    accessToken = response.getJson(dict).get("access_token")
+
+    if not isinstance(accessToken, str):
+        raise TypeError("Something went wrong. Please try again...")
+
+    return accessToken
+
+
+def initializeNodeConfiguration() -> None:
```

Comment:
This function shares a lot of similarities to `node.config` function, why not extract the logic into one function which has the shared functionality part?